codegen/src/test/resources/software/amazon/awssdk/codegen/poet/client/c2j/unsupportedpayloads/service-2.json

```diff
@@ -0,0 +1,2664 @@
+    },
+    "DeliverabilityTestReport":{
+      "type":"structure",
+      "members":{
+        "ReportId":{
+          "shape":"ReportId",
+          "documentation":"<p>A unique string that identifies the predictive inbox placement test.</p>"
+        },
+        "ReportName":{
+          "shape":"ReportName",
+          "documentation":"<p>A name that helps you identify a predictive inbox placement test report.</p>"
+        },
+        "Subject":{
+          "shape":"DeliverabilityTestSubject",
+          "documentation":"<p>The subject line for an email that you submitted in a predictive inbox placement test.</p>"
+        },
+        "FromEmailAddress":{
+          "shape":"EmailAddress",
+          "documentation":"<p>The sender address that you specified for the predictive inbox placement test.</p>"
+        },
+        "CreateDate":{
+          "shape":"Timestamp",
+          "documentation":"<p>The date and time when the predictive inbox placement test was created, in Unix time format.</p>"
+        },
+        "DeliverabilityTestStatus":{
+          "shape":"DeliverabilityTestStatus",
+          "documentation":"<p>The status of the predictive inbox placement test. If the status is <code>IN_PROGRESS</code>, then the predictive inbox placement test is currently running. Predictive inbox placement tests are usually complete within 24 hours of creating the test. If the status is <code>COMPLETE</code>, then the test is finished, and you can use the <code>GetDeliverabilityTestReport</code> to view the results of the test.</p>"
+        }
+      },
+      "documentation":"<p>An object that contains metadata related to a predictive inbox placement test.</p>"
+    },
+    "DeliverabilityTestReports":{
+      "type":"list",
+      "member":{"shape":"DeliverabilityTestReport"}
+    },
+    "DeliverabilityTestStatus":{
+      "type":"string",
+      "documentation":"<p>The status of a predictive inbox placement test. If the status is <code>IN_PROGRESS</code>, then the predictive inbox placement test is currently running. Predictive inbox placement tests are usually complete within 24 hours of creating the test. If the status is <code>COMPLETE</code>, then the test is finished, and you can use the <code>GetDeliverabilityTestReport</code> operation to view the results of the test.</p>",
+      "enum":[
+        "IN_PROGRESS",
+        "COMPLETED"
+      ]
+    },
+    "DeliverabilityTestSubject":{
+      "type":"string",
+      "documentation":"<p>The subject line for an email that you submitted in a predictive inbox placement test.</p>"
+    },
+    "DeliveryOptions":{
+      "type":"structure",
+      "members":{
+        "SendingPoolName":{
+          "shape":"PoolName",
+          "documentation":"<p>The name of the dedicated IP pool that you want to associate with the configuration set.</p>"
+        }
+      },
+      "documentation":"<p>Used to associate a configuration set with a dedicated IP pool.</p>"
+    },
+    "Destination":{
```

Comment:
Do we need all this, or can it be shortened?

Reply:
Shortened.